feature.env

```diff
@@ -1,9 +1,10 @@
 PORT=3007
-HMPPS_AUTH_URL=http://localhost:9091/auth
-TOKEN_VERIFICATION_API_URL=http://localhost:9091/verification
+HMPPS_AUTH_URL=http://127.0.0.1:9091/auth
```

Comment:
docker doesn't seem to play well with localhost on my machine, I did some searches and I think my machine is catered for ipv6 which docker only allows on linux machines so explicitly setting 127.0.0.1 is a workaround for me